### PR TITLE
feature/SOF 7398

### PR DESCRIPTION
**Pull request by VsevolodX**
- feat: add perturbation nb
- chore: a typo

### CODE REVIEW

File: other/materials_designer/create_perturbation.ipynb [hunk 1]
```diff
@@ -0,0 +1,263 @@
+{
```
Comment (timurbazhirov) [inline other/materials_designer/create_perturbation.ipynb:1]: <p>We should demonstrate how to build an arbitrary perturbation here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

**Review by timurbazhirov — APPROVED** [review]

File: other/materials_designer/create_perturbation.ipynb (hunk 1 above)
Comment (VsevolodX) [reply other/materials_designer/create_perturbation.ipynb:1]: Now there's SineWave and arbitrary function examples, do you mean to leave only arbitrary one?

File: other/materials_designer/create_perturbation.ipynb [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
```
Comment (timurbazhirov) [inline other/materials_designer/create_perturbation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #18.        return amplitude * sp.sin(2 * sp.pi * x / wavelength + phase) * sp.sin(2 * sp.pi * y / wavelength)
</pre><p>we should not need to pass <code>function()</code> already executed, we should pass function itself </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (timurbazhirov) [inline other/materials_designer/create_perturbation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #25.    configuration_arbitrary = PerturbationConfiguration(material=supercell,
</pre><p>configuration_custom</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

**Pull request merged**